### ISSUE TITLE
[Enhancement/Request] RS-Clock in RS-Writer

### ISSUE THREAD

**Issue by accessdenied0** [CyclopsMC/IntegratedDynamics#291]
Hello Cyclops-developing-team.

I know it seems trivial and I hope I don't do any request which is/was already made (I looked only up in open issues), but:

I would like to have the RS-Clock in the RS-Writer as well. It makes more sense to me to have a RS-Clock there anyway. "If Boolean is X do RS-Clock Y-Z time" 
X config for true/false
Y ticks pulse on
Z ticks pulse off

I hope you like my request.

**Comment by rubensworks** [comment]
Such a clock already exists in the RS reader, so it already seems trivial to me to connect a reader with a writer in that respect. Is there a certain reason for not doing it that way?

The current way is also a lot more flexible, as you can still transform your clock output variable using a chain of operators.

**Comment by accessdenied0** [comment]
I know, I already mentioned it.

In the current way you have to use/craft a RS reader with actually no other use. You craft an Item with only the use in a network to be a clock. So it is, in fact a "useless" object.

The question is, how it might be more flexible, if it can do the same thing which it is actually supposed to do, emit a signal. 

But indeed I see, now you are mention it, you can't capture the clock as Value from a writer.
Besides that I didn't actually want to move the RS-Clock I further more would like to have a RS-Clock on the Writer, too.

At the end, it is up to you developer if you like it or not. I can't decide what you want in your mod or not, nor I have the right to. I actually find the mod and what you can do with it amazing at it is. I just felt it more of a comfort addition.

But maybe the configurable length of the high and low time is interesting. If not, forget I my idea.

I thank you anyway for reading and considering it.

**Comment by rubensworks** [comment]
You're right, it would be a nice convenience feature. I'll add this to the feature list at a lower priority.

**Comment by accessdenied0** [comment]
I thank you for the work you put into your mods.

**Comment by rubensworks** [comment]
It's my pleasure :-)
